CHANGELOG.md

```diff
@@ -45,6 +45,8 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We fixed an issue where a message about changed metadata would occur on saving although nothing changed. [#9159](https://github.com/JabRef/jabref/issues/9159)
 - When adding or editing a subgroup it is placed w.r.t. to alphabetical ordering rather than at the end. [koppor#577](https://github.com/koppor/jabref/issues/577)
 - We modified the Directory of Open Access Books (DOAB) fetcher so that it will now also fetch the ISBN when possible. [#8708](https://github.com/JabRef/jabref/issues/8708)
+-ISBN-fetcher-for-book-info.com-#9145
+- Add book-into ISBN fetcher to retryFetcher in IsbnFetcher [#9145](https://github.com/JabRef/jabref/issues/9145)
```

Comment:
I think one sentence instead of two is enough for this issue. The change-log is for non-technical people, therefore something like

`We added a new ISBN fetcher (book-info.com), which JabRef will fallback to, if fetching from other ISBN sources will fail [#9145](https://github.com/JabRef/jabref/issues/9145)`